BUILD

```diff
@@ -8,6 +8,7 @@ test_suite(
         ":test_hive_hcatalog",
         ":test_hive_llap",
         ":test_starburst_presto",
+        ":test_spark_rapids",
```

Comment:
It might be `spark-rapids:test_spark_rapids`

Reply:
updated, thanks

Reply:
it also might not?  We're getting the following error:

```
2022-11-14T17:50:31.655759397Z Starting local Bazel server and connecting to it...
2022-11-14T17:50:40.209835763Z ERROR: /init-actions/BUILD:3:11: //:DataprocInitActionsTestSuite: invalid label 'spark-rapids:test_spark_rapids' in element 5 of attribute 'tests' in 'test_suite' rule: invalid target name 'spark-rapids:test_spark_rapids': target names may not contain ':'
2022-11-14T17:50:40.263716460Z ERROR: Skipping ':test_spark_rapids': no such target '//:test_spark_rapids': target 'test_spark_rapids' not declared in package '' defined by /init-actions/BUILD
2022-11-14T17:50:40.304392502Z ERROR: no such target '//:test_spark_rapids': target 'test_spark_rapids' not declared in package '' defined by /init-actions/BUILD
2022-11-14T17:50:40.403247291Z INFO: Elapsed time: 13.751s
2022-11-14T17:50:40.403299411Z INFO: 0 processes.
2022-11-14T17:50:40.403308182Z ERROR: Couldn't start the build. Unable to run tests
```

Reply:
@cjac I update the BUILD file as this [comment](https://github.com/GoogleCloudDataproc/initialization-actions/pull/1018#issuecomment-1256075417) suggested;
and verified the rocky8 patch, the patch worked, could you help build again to see if BUILD file updated correctly?